monkey/monkey_island/cc/services/telemetry/zero_trust_tests/communicate_as_new_user.py

```diff
@@ -0,0 +1,35 @@
+from common.data.zero_trust_consts import EVENT_TYPE_MONKEY_NETWORK, STATUS_FAILED, TEST_COMMUNICATE_AS_NEW_USER, \
+    STATUS_PASSED
+from monkey_island.cc.models.zero_trust.aggregate_finding import AggregateFinding
+from monkey_island.cc.models.zero_trust.event import Event
+
+
+def test_new_user_communication(current_monkey, success, message):
+    tried_to_communicate_event = Event.create_event(
+        title="Communicate as new user",
+        message="Monkey on {} tried to create a new user and communicate from it.".format(current_monkey.hostname),
+        event_type=EVENT_TYPE_MONKEY_NETWORK)
+    events = [tried_to_communicate_event]
+
+    if success:
```

Comment:
Export event creation to function with `success` as param.
I personally would have written the function s.t. `Event.create_event(` is written only once, and the condition controls the value of message.

Reply:
dd9a4b2d

Reply:
+ 76c642e4